tests/models/self_supervised/test_models.py

```diff
@@ -30,6 +28,11 @@ def test_cpcv2(tmpdir, datadir):
         online_ft=True,
         num_classes=datamodule.num_classes,
     )
+
+    # FIXME: workaround for bug caused by
+    # https://github.com/PyTorchLightning/lightning-bolts/commit/2e903c333c37ea83394c7da2ce826de1b82fb356
+    model.datamodule = datamodule
+
     trainer = Trainer(fast_dev_run=True, default_root_dir=tmpdir)
```

Comment:
```suggestion
    trainer = Trainer(fast_dev_run=True, default_root_dir=tmpdir, gpus=1)
```
I do not know lightning that well, but it seems to me, that if it already requires GPU, then it should use it?